app/src/main/kotlin/com/wire/android/ui/calling/ParticipantTile.kt

```diff
@@ -143,6 +144,14 @@ fun ParticipantTile(
 
 @Preview
 @Composable
-@Suppress("EmptyFunctionBlock")
 private fun ParticipantTilePreview() {
+    ParticipantTile(
+        modifier = Modifier.height(300.dp),
```

Comment:
I believe we can use the centralized dimensions here :D

Reply:
This is dynamically generated value, which depends on Phone screen 😅 So I just added random value for preview visualisation 😄

Reply:
Oh sorry didn't catch it was the preview block 😅